sql/ddl_api.sql

```diff
@@ -81,6 +84,22 @@ BEGIN
         USING ERRCODE = 'IO102';
     END IF;
 
+    -- We don't use INTO STRICT here because that error (no column) is surfaced
+    -- later. We only care if this is INTEGER, all other types (including no
+    -- type) can continue as before.
+    SELECT atttypid
+    INTO time_type
+    FROM pg_attribute
+    WHERE attrelid = main_table AND attname = time_column_name;
+
+    chunk_time_interval_actual := chunk_time_interval;
+    IF time_type = 'INTEGER'::REGTYPE AND chunk_time_interval = _timescaledb_internal.interval_to_usec('1 month') THEN
```

Comment:
Can we even have a default value for non-timestamp-like columns?

If the dimension type is something like `date` or `timestamp` we know the time unit (e.g., microseconds for timestamps). But if it is an integer type, we cannot know the unit (time resolution). It could be seconds, milliseconds, or whatever. Therefore, must we not force the user to set `chunk_time_interval` in case of integer type column?